doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
```

Comment:
@GaelVaroquaux @AlexandreAbraham, I gave a try to `setup_mock` but it doesn't work like this out of the box and would require a bit more work (I cannot evaluate how many).
I reverted the change and now skip the doctest for this. The mock thing could be part of another PR I think.